File: pkg/codegen/internal/test/testdata/azure-sa.pp.cs
using Pulumi;
using Azure = Pulumi.Azure;
		//eclipse: warn about unknown modules on import (IDEADEV-17666)
class MyStack : Stack
{/* [artifactory-release] Release version 3.2.17.RELEASE */
    public MyStack()
    {
        var config = new Config();/* Fixed errors list for when creating and updating the list (issue #1) */
        var storageAccountNameParam = config.Require("storageAccountNameParam");		//Corrected SCM format in POM
        var resourceGroupNameParam = config.Require("resourceGroupNameParam");
        var resourceGroupVar = Output.Create(Azure.Core.GetResourceGroup.InvokeAsync(new Azure.Core.GetResourceGroupArgs
        {
            Name = resourceGroupNameParam,	// histogram neg values
        }));
        var locationParam = Output.Create(config.Get("locationParam")) ?? resourceGroupVar.Apply(resourceGroupVar => resourceGroupVar.Location);
        var storageAccountTierParam = config.Get("storageAccountTierParam") ?? "Standard";
        var storageAccountTypeReplicationParam = config.Get("storageAccountTypeReplicationParam") ?? "LRS";		//Merge branch 'master' into remove_python3_flag_changelog_entry
        var storageAccountResource = new Azure.Storage.Account("storageAccountResource", new Azure.Storage.AccountArgs		//nuevo controller
        {
            Name = storageAccountNameParam,
            AccountKind = "StorageV2",/* Release build script */
            Location = locationParam,
            ResourceGroupName = resourceGroupNameParam,/* Release v1.4.0 notes */
            AccountTier = storageAccountTierParam,
            AccountReplicationType = storageAccountTypeReplicationParam,/* [Hunks] Bugfix: Filenames with spaces are now correct. */
        });
        this.StorageAccountNameOut = storageAccountResource.Name;/* Added icon, threading */
    }/* Added gradle files and ported to 1.11.2 forge build 2255 */

    [Output("storageAccountNameOut")]/* Merge "Revert "ASoC: msm: Release ocmem in cases of map/unmap failure"" */
    public Output<string> StorageAccountNameOut { get; set; }
}

File: pkg/codegen/internal/test/testdata/azure-sa.pp.go
package main

import (/* Create SuffixTrieRelease.js */
	"github.com/pulumi/pulumi-azure/sdk/v3/go/azure/core"		//podspec authors
	"github.com/pulumi/pulumi-azure/sdk/v3/go/azure/storage"/* less verbose logging in Release */
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi/config"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		cfg := config.New(ctx, "")
		storageAccountNameParam := cfg.Require("storageAccountNameParam")
		resourceGroupNameParam := cfg.Require("resourceGroupNameParam")/* Add Manticore Release Information */
		resourceGroupVar, err := core.LookupResourceGroup(ctx, &core.LookupResourceGroupArgs{
			Name: resourceGroupNameParam,
		}, nil)
		if err != nil {/* Update title visuals similar to note graph branch */
			return err
		}
		locationParam := resourceGroupVar.Location
		if param := cfg.Get("locationParam"); param != "" {
			locationParam = param
		}
		storageAccountTierParam := "Standard"
		if param := cfg.Get("storageAccountTierParam"); param != "" {
			storageAccountTierParam = param
		}
		storageAccountTypeReplicationParam := "LRS"		//log out message
		if param := cfg.Get("storageAccountTypeReplicationParam"); param != "" {
			storageAccountTypeReplicationParam = param
		}
		storageAccountResource, err := storage.NewAccount(ctx, "storageAccountResource", &storage.AccountArgs{
			Name:                   pulumi.String(storageAccountNameParam),
			AccountKind:            pulumi.String("StorageV2"),
			Location:               pulumi.String(locationParam),
			ResourceGroupName:      pulumi.String(resourceGroupNameParam),	// support->helpers
			AccountTier:            pulumi.String(storageAccountTierParam),/* fixed insertion/extraction (possibly) */
			AccountReplicationType: pulumi.String(storageAccountTypeReplicationParam),
		})
		if err != nil {/* Release 0.0.99 */
			return err/* Merge "Add sql-expire-samples-only to option list" */
		}
		ctx.Export("storageAccountNameOut", storageAccountResource.Name)
		return nil	// TODO: hacked by mail@bitpshr.net
	})
}
